packages/ui-library/src/components/styled/index.tsx

```diff
@@ -27,3 +28,8 @@ export const ColorCircle = styled.div<{ color?: string; size?: number }>`
     display: inline-block
   `}
 `;
+
+export const StyledPageTitleDescription = styled(Typography).attrs({ variant: 'bodyLargeRegular' })`
+  color: ${({ theme: { palette } }) => colors[palette.mode].typography.typo2};
```

Comment:
We sure about this one? Lots of those had typo1 defined into it

Reply:
They should all be typo2!